tutorials/plugins/running_code_in_the_editor.rst

```diff
@@ -245,6 +245,147 @@ angle add a setter ``set(new_speed)`` which is executed with the input from the
     to run in the editor too. Autoload nodes cannot be accessed in the editor at
     all.
 
+Getting notified about Resources changing
+-----------------------------------------
+On some occastions you want your tool to use a resource. However, when you change an
+attribute of that resource in the editor, the ``set()`` function of your tool will
+not get called.
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    @tool
+    class_name MyTool
+    extends Node
+
+    @export var resource:MyResource:
+        set(new_resource):
+            resource = new_resource
+            _on_resource_set()
+    
+    # This will only be called, when you create, delete or paste a resource.
+    # You will not get an update when tweaking properteis of it.
```

Comment:
```suggestion
    # This will only be called when you create, delete, or paste a resource.
    # You will not get an update when tweaking properties of it.
```